.github/workflows/clang.yaml

```diff
@@ -0,0 +1,163 @@
+name: Build with Clang/LLVM
+on:
+  push:
+    branches:
+      - main
+      - v*-branch
+      - collab-*
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  clang-build:
+    if: github.repository_owner == 'zephyrproject-rtos'
+    runs-on:
+      group: test-runner-v2-linux-x64-4xlarge
```

Comment:
Are they ready for prime time? `test` in the name kind of suggests otherwise so just checking :)

Reply:
oops, good catch